src/RealMeService.php

```diff
@@ -2,9 +2,11 @@
 
 namespace SilverStripe\RealMe;
 
+use ArgumentCountError;
```

Comment:
This is PHP 7 only

Reply:
Haha I used it, realised it was 7.1 and then I removed it - but I forgot to remove the import >.<